frontend/app/enrollment/task/[taskId]/page.tsx

```diff
@@ -18,46 +20,93 @@ export default function EnrollmentTaskPage() {
         }
     }, [taskId, setSelectedTaskId])
 
-    if (loading || !initialized) return <Loading />
+    if (loading || !initialized) return <Loading/>
 
-    const serviceName = task?.focus?.display || "Unknown"
-    const conditionName = task?.reasonCode?.coding?.[0].display || "Unknown"
-    const taskDate = task?.meta?.lastUpdated ? new Date(task.meta.lastUpdated).toLocaleDateString("nl-NL") : "Onbekend"
+    if (!task) {
+        return <div className='w-[568px] flex flex-col gap-4'>Ongeldige taak</div>
+    }
+
+    const StatusElement = (props: { label: string, value: string }) =>
+        <>
+            <div className="text-muted-foreground">{props.label}:</div>
+            <div className="font-bold">{props.value}</div>
+        </>
```

Comment:
Usually arguments are directly loaded via "object destructuring":

```suggestion
    const StatusElement = ({label, value}: { label: string, value: string }) =>
        <>
            <div className="text-muted-foreground">{label}:</div>
            <div className="font-bold">{value}</div>
        </>
```

Reply:
Cheers, learned something new.